pkg/kev/converter/kubernetes/transform.go

```diff
@@ -1684,8 +1684,8 @@ func (k *Kubernetes) updateKubernetesObjects(projectService ProjectService, obje
 	// @step configure capabilities
 	capabilities := k.configCapabilities(projectService)
 
-	// @step configure annotations
-	annotations := configAnnotations(projectService)
+	// @step configure podAnnotations
```

Comment:
small change `podAnnotations` -> `pod annotations`

Reply:
Actually this should just say `annotations` as we don't configure pod annotations  here.